packages/block-editor/src/components/colors-gradients/dropdown.js

```diff
@@ -143,8 +188,12 @@ export default function ColorGradientSettingsDropdown( {
 					...setting,
 				};
 				const toggleSettings = {
-					colorValue: setting.gradientValue ?? setting.colorValue,
+					clearable: controlProps.clearable,
```

Comment:
```suggestion
					clearable: setting.clearable,
```

No strong opinion, but it might be better to align it with the format of the other properties.